### ISSUE TITLE
When Blaze Form contains a CRUD it inserst REACT code to the page JS

### ISSUE THREAD

**Issue by adnoh** [perak/kitchen-site#379]
If I have a form with a crud it generates me for my blaze page.js file
```
Template.UserSettingsSkillsEditAllgemeinEditFormCommon.onRendered(function() {
  pageSession.set("commonSprachenCrudItems", this.props.data.current_user_profile.commonSprachen || []);
```
**this.props.data** looks like react code to me-blaze uses just **this.data**. The generated code does not run - so my CRUD Fields are not show in the page template (Helper for: crud-table-header)

**Comment by perak** [comment]
Ooops, fixed and will be deployed in next version v0.9.82

**Comment by perak** [comment]
Fixed and deployed.
